store/sqlstore/upgrade.go

```diff
@@ -1129,136 +1138,62 @@ func upgradeDatabaseToVersion535(sqlStore *SqlStore) {
 				q.UserId = ChannelMembers.UserId AND
 				ChannelMembers.MentionCount > 0
 		`
-		}
-		if _, err := sqlStore.GetMaster().ExecNoTimeout(updateMentionCountRootQuery); err != nil {
-			mlog.Error("Error updating ChannelId in Threads table", mlog.Err(err))
-		}
-		sqlStore.CreateColumnIfNotExists("Channels", "TotalMsgCountRoot", "bigint", "bigint", "0")
-		sqlStore.CreateColumnIfNotExistsNoDefault("Channels", "LastRootPostAt", "bigint", "bigint")
-		defer sqlStore.RemoveColumnIfExists("Channels", "LastRootPostAt")
+	}
+	if _, err := sqlStore.GetMaster().ExecNoTimeout(updateMentionCountRootQuery); err != nil {
+		mlog.Error("Error updating ChannelId in Threads table", mlog.Err(err))
```

Comment:
Gentle reminder to check our logging [principles](https://developers.mattermost.com/contribute/server/style-guide/#log-levels) before merging this change.